commands/moderation/filter.js

```diff
@@ -0,0 +1,113 @@
+/*  
+  List of offensive words for easy readability
+  
+    Strong Words 
+      *one of these words will trigger the filter 
+       hell          damn
+       goddamn       goddamned
+       goddamnit     damned
+       damnit        hell
+       fuck          fucked
+       fucking       fucker
+       fuckwad       buttplug
+       motherfucker  wtf
+       nigger        nigga
+       penis         vagina
+       asshole       sex
+       sexed         piss
+       pissed        sexy
+       sexual        sexuality
+       bastard       clitoris
+       bitch         boobs
+       semen         sperm
+       jizzed        jizz
+       whore         prostitute
+       fornicate     fornication
+       adultery      adulter
+       adulteress    slut
+       condom        shit
+       
+    Minor Words    
+       *3+ of these words will trigger the filter
+       darn        dern
+       darned      derned
+       darnit      dernit
+       dang        heck
+       frick       fricked
+       crap        omg
+      
+*/
+
+const Discord = require('discord.js');
+
+module.exports = {
+  name: "filter",
+  description: "filter a message",
+
+  execute(msg) {
+    // Make sure user should be filtered
+    if (
+    !msg.member.roles.cache.some(
+      (role) =>
+        role.name === 'Super User' ||
+        role.name === 'Moderator' ||
+        role.name === 'Admin'
+    )
```

Comment:
I'm starting to think we should separate out a version or two of @fedGL's `isHighRoller` function into its own module so that we can just import it and use it where we need.  Right now I think something similar to this code is defined in almost every command file, which is the opposite of the DRY principle...

That should probably be done as a separate refactoring issue.

Reply:
I went ahead and made a function like this in the filter.js file so it will be easier to refactor in the future.